lib/messages/deployment-status.js

```diff
@@ -10,24 +10,30 @@ class DeploymentStatus extends Message {
         htmlURL: deployment.creator.html_url,
       },
       footer: `<${repository.html_url}|${repository.full_name}>`,
-
     });
     this.deploymentStatus = deploymentStatus;
     this.deployment = deployment;
     this.repository = repository;
-    this.shortSha = this.deployment.sha.substr(0, 7);
     this.commitLink = `${this.repository.html_url}/commit/${this.deployment.sha}`;
   }
 
+  prettyRef() {
```

Comment:
If we prefix this method definition with `get` then we don't need to always call it with `()` further down below.

```suggestion
  get prettyRef() {
```

Prior art: https://github.com/integrations/slack/blob/fad9c6409bf720b3b36bae3b1e15ad73dc7fdfc4/lib/messages/flow/updated-settings.js#L10-L28

Totally just an idea though, feel free to ignore